website/docs/tutorials/06-deploying/03-network_config.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_label: Network config
+---
+# Network config
+
```

Comment:
please use https://peps.python.org/pep-0589/ to define a type `NetworkConfig` and use it in all snippets in this PR, i.e. `config: NetworkConfig`